docs/source/userguide/mesh.rst

```diff
@@ -312,3 +312,250 @@ Meshes from FEniCS
 ------------------
 
 See the `FEniCS documentation <https://fenicsproject.org/olddocs/dolfin/latest/python/demos/built-in-meshes/demo_built-in-meshes.py.html>`_ for more built-in meshes.
+
+------------------
+Meshes from GMSH
+------------------
+
+The Python API of GMSH can be used to create meshes that are useable in FESTIM. Here we will walk through its usage when creating a monoblock subsection consisting of tungsten surrounding a tube of CuCrZr
+
+<img width="948" alt="Screenshot 2024-09-11 at 17 09 10" src="https://github.com/user-attachments/assets/1ec52a67-840d-4ef8-a741-a79f0e2c1cc3">
+
+Firstly, GMSH must be imported and initialised.
+
+```
+import gmsh as gmsh
+
+gmsh.initialize()
+gmsh.model.add("mesh")
+```
```

Comment:
For code blocks, please use the same syntax as elsewhere in this file:

```
.. code-block:: python
   
    your code goes here with an indent
```

This is not needed for inline code.